examples/repo_org_internal/cli.cfg

```
@@ -0,0 +1,10 @@
+provider_installation {
+  filesystem_mirror {
+    path    = "/Users/jurgen.weber/checkouts/terraform/terraform-provider-github/examples/repo_org_internal/terraform.d/plugins"
+    include = ["*/*/*"]
+  }
+  direct {
+    exclude = ["*/*/*"]
+  }
+}
```

```suggestion
See Building The Provider instructions in CONTRIBUTING.md
```

Can we make this more general by pointing to docs in the repo?

---

Definitely! Would you prefer that we keep this `cli.cfg` file the pointer to the repo docs, or maybe should we add a link to the README in that directory? It feels a little weird to me to keep the `.cfg` extension for a text or markdown file...what do you think?

---

> add a link to the README in that directory

Good catch!  My vote goes to adding a link.  https://github.com/integrations/terraform-provider-github/blob/master/CONTRIBUTING.md#building-the-provider addresses this indirectly, so maybe we could link to there.

---

Done! I've removed the config and provided the link [here](https://github.com/integrations/terraform-provider-github/pull/746/commits/b94f595dca2dd56a6601e275d7feef31fbbd9e2e), and I've also renamed the example directory and created repo name to match the existing pattern [in this commit](https://github.com/integrations/terraform-provider-github/pull/746/commits/303db1f3e317e0bb9b17fd28029d13b3a1c93d00). 

How does that look?

---

Looking great 🚀